### PR TITLE
Added pico-w blink example

### PR DESCRIPTION
**Pull request by yochidros**
Hi! I tried using the [pico-blink-sdk](https://github.com/apple/swift-embedded-examples/tree/main/pico-blink-sdk) with a Pico-W, but it didn't work because the LED module is on the wireless chip. For the Pico-W, the pico-sdk doesn't define PICO_DEFAULT_LED_PIN; instead, it uses CYW43_WL_GPIO_LED_PIN.
Ref: [Pico-SDK](https://github.com/raspberrypi/pico-sdk/blob/6a7db34ff63345a7badec79ebea3aaef1712f374/src/boards/include/boards/pico_w.h#L31C15-L32C37)

I also tried using CYW43_WL_GPIO_LED_PIN to make the Pico-W blink. By linking several necessary modules, I was able to get it working successfully!

I thought this example might be useful to other developers, so I added it!

I recently close PR #6, but @kubamracek commented https://github.com/apple/swift-embedded-examples/pull/6#issuecomment-2106657865. I appreciate that. I added pico-w-blink-sdk entry into the top-level README.

### CODE REVIEW

File: pico-w-blink-sdk/BridgingHeader.h [hunk 1]
```diff
@@ -0,0 +1,15 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors.
```
Comment (rauhul) [inline pico-w-blink-sdk/BridgingHeader.h:5]: Could you update the year to 2024 for the files added?

**Comment by kubamracek** [comment]
Could you scale down the photo of the board? (it’s now an 8 MB jpeg)

File: pico-w-blink-sdk/BridgingHeader.h (hunk 1 above)
Comment (yochidros) [reply pico-w-blink-sdk/BridgingHeader.h]: @rauhul I fixed it. https://github.com/apple/swift-embedded-examples/pull/11/commits/ba8f899073e86b0e49c145c113d0d9c991df4122

**Comment by yochidros** [comment]
@kubamracek 
I scaled down the photo 8MB ->  691KB!!

**Comment by kubamracek** [comment]
Looks great, and I confirmed it works on a Pico W. Thanks!!!

**Review by kubamracek — APPROVED** [review]

**Comment by unnamedd** [comment]
I really thank you all for the hard work done in this repo and also in this repo! I do have some Picos at home and I really want to get an extra time for playing with it.

Thanks a lot guys! You are awesome!

**Comment by yochidros** [comment]
@kubamracek @rauhul 
thank you for review.
I can't merge this PR. so, please could you merge it when you have time.🙏

**Comment by kubamracek** [comment]
Thanks @yochidros, this is a great contribution :)

**Pull request merged**